### ISSUE TITLE
Training cost remains high

### ISSUE THREAD

**Issue by guozixunnicolas** [qiuqiangkong/sampleRNN_acoustic_scene_generation#9]
Hi Qiuqiang,

Thank you for your brilliant work! Im currently trying to re-implement the whole thing in tensorflow. 

However my training loss remains high( around 2 ). I've noticed that the training/validation/test cost is around 2.7 in your implementation. I wonder can the model produce useful result with such high loss value?

If it's convenient for you, would you mind sharing with me a screenshot of training loss curve? 

Many thanks!

Best regards,

ZX

**Comment by qiuqiangkong** [comment]
Hi ZX,

    We did not keep the training curve in our disk. For acoustic scene generation, we observed higher loss (around 2) than generating music and speech (around 1.2). I guess it is because of the randomness in acoustic scenes. Though the loss is high, the system can still generate acoustic scenes.


Best wishes,


Qiuqiang

________________________________

**Comment by guozixunnicolas** [comment]
Hi Qiuqiang,

Thank you for your answer! That helps:)

Have a  nice day!

Best regards,

ZX

**Comment by guozixunnicolas** [comment]
@qiuqiangkong Hi qiuqiang, just a short follow-up question, may I know how long does it take for the model to generate 1 second of audio? Mine needs around 6hour(with a GPU) to generate 1 seconds of audio which seems to be wrong. Thanks in advance:)
